ios/Podfile.lock

```diff
@@ -1751,4 +1751,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: 407c2bad1f5a0b2a39e0f4ece4251ac7baf4239e
 
-COCOAPODS: 1.14.3
+COCOAPODS: 1.15.2
```

Comment:
Please revert the changes.

To make sure everyone works with the same Cocoapods version please install bundler:
https://omurbilgili.medium.com/unveiling-the-power-of-metro-bundler-in-react-native-bf68d151f063

Then you can ecexute `yarn pods` command which is essentially `bundle exec pod install --project-directory=ios` command under the hood.